tests/lib/naming_test.py

```diff
@@ -169,5 +171,123 @@ def testParseServiceFile(self):
     filedefs._ParseFile(data, 'services')
     self.assertEqual(filedefs.GetService('HTTP'), ['80/tcp'])
 
+
+class Naming_Characterization_Tests(unittest.TestCase):
```

Comment:
Sorry I am not exactly sure what you mean? Are you talking about making tests that use NETWORK.def or some file? I feel that naming.py is pretty straight forward in its use and does not really require this. If you would like to rework documentation that is fine but I don't believe our documentation is lacking in this regard.

https://github.com/google/capirca/wiki/Naming-definitions
https://github.com/google/capirca/wiki/Naming-library
https://github.com/google/capirca/wiki/Policy-format

Between those three pages it decribes the naming format, the methods, and how they are used in policies.